src/shared/components/footer/footer.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="footer">
+  <div class="footer-info">
+    <svg-icon [src]="'shoperz-logo'"></svg-icon>
+    <div>
+      <h5 class="footer-info-title">headquarters</h5>
+      <p>Presei Square No45, Bucharest - 099455</p>
+    </div>
+    <div>
+      <h5 class="footer-info-title">email</h5>
+      <p>contact@shoperz.com</p>
+    </div>
+    <div>
+      <h5 class="footer-info-title">telephone</h5>
+      <p>(+40) 987 123 654</p>
+    </div>
+    <div class="footer-info-icons">
+      <a class="footer-info-icon"
+         href="https://www.facebook.com/"
+         target="_blank">
+        <svg-icon [src]="'facebook-logo'"></svg-icon>
+      </a>
+      <a class="footer-info-icon"
+         href="https://www.instagram.com/"
+         target="_blank">
+        <svg-icon [src]="'instagram-logo'" class="mr-6"></svg-icon>
+      </a>
+      <a class="footer-info-icon"
+         href="https://www.linkedin.com/"
+         target="_blank">
+        <svg-icon [src]="'linkedin-logo'" class="mr-6"></svg-icon>
+      </a>
+      <a class="footer-info-icon"
+         href="https://twitter.com/?lang=en"
+         target="_blank">
+        <svg-icon [src]="'twitter-logo'" class="mr-6"></svg-icon>
+      </a>
+      <a class="footer-info-icon"
+         href="https://www.youtube.com/"
+         target="_blank">
+        <svg-icon [src]="'youtube-logo'" class="mr-6"></svg-icon>
+      </a>
+    </div>
+  </div>
+
+  <div class="footer-categories">
+    <p class="footer-categories-title">categories</p>
+    <ul>
+      <li *ngFor="let link of categoriesLinks"
+          [queryParams]="link.queryParams"
+          [routerLink]="link.routerLink"
+          class="footer-links">
+        {{ link.title }}
+      </li>
+    </ul>
+
+  </div>
+  <div class="footer-useful-links">
+    <p class="footer-useful-links-title">useful links</p>
+    <ul>
+      <li *ngFor="let link of usefulLinks"
+          class="footer-links">
+        <a routerLink="">{{ link }}</a>
+      </li>
+    </ul>
+  </div>
+  <div class="footer-customer-service">
+    <p class="footer-customer-service-title">customer service</p>
+    <ul>
+      <li *ngFor="let link of customerServiceLinks"
+          class="footer-links">
+        <a routerLink="">{{ link }}</a></li>
+    </ul>
+  </div>
+</div>
+
+<div class="bottom-footer">
+  <p>&copy; <strong>Shoperz</strong> {{ year }} - All Rights Reserved</p>
```

Comment:
Use span & tailwind to make the font bold.
The <strong> tag is not predictable and we cannot know what the font-weight is here.
It can be displayed differently in browsers and can not correspond to the customer's Figma design.